### PR TITLE
Sites controller

### PR DESCRIPTION
**Pull request by barmintor**

### CODE REVIEW

File: config/initializers/subsites.rb [hunk 1]
```diff
@@ -1 +1,34 @@
-SUBSITES = YAML.load_file("#{Rails.root.to_s}/config/subsites.yml")[Rails.env]
\ No newline at end of file
+SUBSITES = YAML.load_file("#{Rails.root.to_s}/config/subsites.yml")[Rails.env].with_indifferent_access
+SUBSITES['sites'] = {}.with_indifferent_access
```
Comment (elo2112) [inline config/initializers/subsites.rb]: Is SUBSITES['sites'] still necessary?

Comment (barmintor) [inline config/initializers/subsites.rb]: used in the sites_controller to identify the publisher of Concepts for the index action.

Comment (elo2112) [inline config/initializers/subsites.rb]: Could you link me to the line where that happens?

Comment (elo2112) [inline config/initializers/subsites.rb]: Update: Deleting that SUBSITES['sites'] line now in a new commit, as per our Slack conversation.

**Pull request merged**